scripts/zephyr_module.py

```diff
@@ -257,27 +257,32 @@ def process_blobs(module, meta):
     return blobs
 
 
-def kconfig_snippet(meta, path, kconfig_file=None):
+def kconfig_snippet(meta, path, kconfig_file=None, blobs=False):
     name = meta['name']
     name_sanitized = meta['name-sanitized']
 
-    snippet = (f'menu "{name} ({path.as_posix()})"',
+    snippet = [f'menu "{name} ({path.as_posix()})"',
                f'osource "{kconfig_file.resolve().as_posix()}"' if kconfig_file
                else f'osource "$(ZEPHYR_{name_sanitized.upper()}_KCONFIG)"',
                f'config ZEPHYR_{name_sanitized.upper()}_MODULE',
                '	bool',
                '	default y',
-               'endmenu\n')
+               'endmenu\n']
+
+    if blobs:
+        snippet.insert(-1, '	select TAINT_BLOBS')
```

Comment:
instead of manipulating array post creation, why not do like:
```
    snippet = [f'menu "{name} ({path.as_posix()})"',
               f'osource "{kconfig_file.resolve().as_posix()}"' if kconfig_file
               else f'osource "$(ZEPHYR_{name_sanitized.upper()}_KCONFIG)"',
               f'config ZEPHYR_{name_sanitized.upper()}_MODULE',
               '	bool',
               '	default y',
               '	select TAINT_BLOBS' if blobs else '',
               'endmenu\n']
```
This of course will give a blank line in the genercated Kconfig on all modules that doesn't TAINT, but can easily be addressed by moving the `\n` up and remove it in the `.join()` call.
Resulting in:
```suggestion
    snippet = [f'menu "{name} ({path.as_posix()})"\n',
               f'osource "{kconfig_file.resolve().as_posix()}"\n' if kconfig_file
               else f'osource "$(ZEPHYR_{name_sanitized.upper()}_KCONFIG)"\n',
               f'config ZEPHYR_{name_sanitized.upper()}_MODULE\n',
               '	bool\n',
               '	default y\n',
               '	select TAINT_BLOBS\n' if blobs else '',
               'endmenu\n']
```

Reply:
Will let you improve this in a follow-up PR, along with support for modules that do not have Kconfig files.